build.gradle

```diff
@@ -94,7 +94,7 @@ task jar (type: Jar, overwrite: true) {
         exclude 'freenet/node/Version.class'
         exclude 'freenet/node/Version$1.class'
     }
-    preserveFileTimestamps = false
+    preserveFileTimestamps = true
```

Comment:
What's the reasoning for setting this to `true`? For deterministic builds it needs to be `false` - [documentation of that is here](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Zip.html#org.gradle.api.tasks.bundling.Zip:preserveFileTimestamps). If there's a need that it not be deterministic, I think we'd need another `task` to build for the Debian package.